smoke-tests-otel-starter/spring-boot-common/src/main/resources/application.yaml

```diff
@@ -1,5 +1,8 @@
 otel:
   instrumentation:
+    common:
+      db-statement-sanitizer:
+        enabled: false
```

Comment:
if we're only going to test one state (sanitizer on or off), I'd prefer to test sanitizer on given the high cost if that particular state breaks

Reply:
on is the default - the purpose of the test is to test the wiring

Reply:
do we have another test that ensures the wiring code doesn't break the default?

Reply:
no - I can create one later if needed

Reply:
> if we're only going to test one state (sanitizer on or off), I'd prefer to test sanitizer on given the high cost if that particular state breaks

it would be a really bad CVE filed against us if the default state breaks

Reply:
@trask done